backend/api/Areas/Property/Controllers/SearchController.cs

```diff
@@ -2,7 +2,6 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http.Extensions;
 using Microsoft.AspNetCore.Mvc;
-using NetTopologySuite.IO;
```

Comment:
this is unrelated right? I wonder if we could get basic static analysis going for .net at some point.

Reply:
Unrelated, but an unused `using`.  It would be helpful to have more linting on the backend, but I don't have time to address.